Examples/React/BUILD

```diff
@@ -1,14 +1,41 @@
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_application")
+load("@rules_cc//cc:defs.bzl", "objc_library")
+
 objc_library(
-    name="all",
-    deps=[
-        "//Vendor/React:React-Core",
-        "//Vendor/React:DevSupport",
-        "//Vendor/React-RCTImage:React-RCTImage",
-        "//Vendor/React-RCTNetwork:React-RCTNetwork",
-        "//Vendor/React-RCTText:React-RCTText",
-        "//Vendor/React-Core:RCTWebSocket",
-        "//Vendor/React-CoreModules:React-CoreModules",
-        "//Vendor/React-RCTAnimation:React-RCTAnimation",
-        "//Vendor/ReactCommon:turbomodule_core",
-        ]
+    name = "AppClasses",
+    srcs = [
+        "App/AppDelegate.m",
+        "App/main.m",
+    ],
+    hdrs = glob(["App/*.h"]),
+    sdk_frameworks = [
+        "QuartzCore",
+        "CFNetwork",
+        "CoreGraphics",
+        "ImageIO",
+        "Security",
+        "Accelerate",
+    ],
+    deps = [
+        "//Vendor/React",
+        "//Vendor/React:React_hmap",
+    ],
+    copts = [
+        "-I$(GENDIR)/Vendor/React/React_hmap.hmap",
+        "-I."
+    ],
+)
+
+ios_application(
```

Comment:
@jerrymarino I expended the React example to actually build the iOS app because there are a couple gotchas in there that took me some time to figure out:
1. The 2016 version of Folly doesn't compile, and the newer v2020.01.13.00 requires an odd workaround
2. You have to use the React_hmap in order for your application code to import from `<React/Whatever>`